conformance-tests/build.gradle

```diff
@@ -40,6 +40,9 @@ distributions {
                 from depsJar
                 from configurations.assertionsRuntimeClasspath
             }
+            into('/samples') {
+                from '../samples'
```

Comment:
Should this also be put into the `sourceSets` to have everything defined there?

Reply:
I don't want to move the samples at this time, since others might need them to be where they are.

And I don't want to compile them, since some of them may have compilation errors. For instance, many of them depend on `@NullUnspecified`, which doesn't actually exist.

Eventually, I'd like to see that directory deleted, once all the use cases are expressed in the real conformance tests.